frontend/public/components/container-selector.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import { ContainerSpec } from '../module/k8s';
+import { Checkbox } from '@patternfly/react-core';
+
+export const ContainerSelector: React.FC<ContainerSelectorProps> = ({ containers, onChange, selected }) => <div>
+  {containers.map((container: ContainerSpec) => <Checkbox
+    key={container.name}
+    label={`${container.name} from image ${container.image}`}
+    id={container.name}
+    isChecked={selected.includes(container.name)}
+    checked={selected.includes(container.name)}
```

Comment:
Do you mean to have both `isChecked` and `checked`?

Reply:
No, only `isChecked` was needed.  That was a relic from when the checkbox was not getting updated.